packages/rum-core/src/rawRumEvent.types.ts

```diff
@@ -41,6 +41,7 @@ export interface RawRumResourceEvent {
     span_id?: string // not available for initial document tracing
     rule_psr?: number
     discarded: boolean
+    resolveDuration?: ServerDuration
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏is it only for while we are monitoring the change or do we plan to keep this property?